lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -58,6 +58,21 @@ def self.symbol_to_cli(symbol)
     end
   end
 
+  # General utility class
```

Comment:
I think this may have been a bad merge - these two methods should belong inside the other "class Utils" below, rather than having two different declarations of this class. I fixed this in grpc fairly recently.